packages/node_modules/nav-frontend-skjema/src/radio-panel-gruppe.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { SkjemaGruppe, Fieldset } from './';
+import 'nav-frontend-skjema-style';
+
+export interface RadioProps {
+    label: string;
+    value: string;
+    id: string;
+    disabled?: boolean;
+}
+
+export interface FeilProps {
+    feilmelding: React.ReactNode | React.ReactChild | React.ReactChildren;
+}
+
+export interface RadioPanelGruppeProps {
+    radios: RadioProps[];
+    name: string;
+    legend: string;
+    onChange: (v: string) => void;
+    checked?: string;
+    feil?: FeilProps;
+}
+
+export interface RadioPanelProps extends RadioProps {
+    checked: boolean;
+    name: string;
+    onChange: () => void;
+}
+
+interface RadioPanelState {
+    hasFocus: boolean;
+}
+
+class RadioPanel extends React.Component<RadioPanelProps, RadioPanelState> {
+    constructor(props: RadioPanelProps) {
+        super(props);
+        this.state = { hasFocus: false };
+    }
+
+    toggleOutline() {
+        this.setState({ hasFocus: !this.state.hasFocus });
+    }
+
+    render() {
+        const { checked, disabled, label, name, id, onChange } = this.props;
+        const { hasFocus } = this.state;
+
+        const cls = classNames('radioPanel', {
+            'radioPanel--checked': checked === true && !disabled,
+            'radioPanel--focused': hasFocus === true && !disabled,
+            'radioPanel--disabled': disabled === true
+        });
+
+        return (
+            <div
+                className={cls}
+                onClick={() => {
```

Comment:
`onClick` på ett ikke-interaktivt element (e.g div) er litt rart. Egentlig rart at eslint ikke reagerer på det. 
Men kunne noe alà dette fungere her:
```
<label>
  <input type="radio" />
  <span>{label}</span>
</label>
```

Reply:
Skal prøve å få den til å funke. Vet du om det isåfall fremdeles blir riktig å putte htmlFor på `<label>`, selv om `input type=radio` ligger wrappet i labelen og teksten ligger i et `span`-child på labelen?

Reply:
Etter det jeg et så er det ikke nødvendig med `htmlFor` når label-elementet ligger rundt selve input-elementet.